recipes/glib/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
+
+package:
+  name: glib
+  version: {{ version }}
+
+source:
+  fn: glib-{{ version }}.tar.xz
+  url: http://ftp.acc.umu.se/pub/GNOME/sources/glib/{{ major_minor }}/glib-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+#  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    #- gcc  # [linux]
+    #- gcc  # [osx]
+    - python
+    - pkg-config
+    - libffi
+    - gettext
+    - zlib
+    - pcre
+    - libiconv
```

Comment:
It probably should be as we are decompressing an `xz` file. Not sure how we are getting away with not having it, but it should be added.